pkg/image/controller/trigger/image_trigger_controller_test.go

```diff
@@ -153,43 +154,51 @@ type fakeBuildConfigInterface struct {
 	namespace string
 }
 
-func (fakeBuildConfigInterface) Create(ctx context.Context, bc *buildv1.BuildConfig, opts metav1.CreateOptions) (*buildv1.BuildConfig, error) {
+func (*fakeBuildConfigInterface) Apply(context.Context, *buildapply.BuildConfigApplyConfiguration, metav1.ApplyOptions) (*buildv1.BuildConfig, error) {
```

Comment:
@sanchezl just a nit: worth keeping the same order of methods to reduce the diff

Reply:
I didn't re-order the methods (just added some), but there were enough changes[^1] to the existing methods that the diff does not match them up.


[^1]: Consistently used pointer receivers. Cleaned up unused parameters names.